amazon-chime-sdk-js/src/transceivercontroller/DefaultTransceiverController.ts

```diff
@@ -574,6 +649,120 @@ export default class DefaultTransceiverController
     this.addRedundantAudioRecoveryMetricsObserver(this.meetingSessionContext?.statsCollector);
   }
 
+  protected setupInsertableStreamWorker(): void {
```

Comment:
It would be great to move this function to `setupSenderInsertableStream` and use it instead. So that you have one to handle sender and the other one to handle receiver.